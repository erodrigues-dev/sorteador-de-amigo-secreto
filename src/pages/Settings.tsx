import React from 'react'
import { Footer } from '../components/Footer'
import Form from '../components/Form/Form'
import { List } from '../components/List'

export const Settings = () => {
  return (
    <>
      <Form />
      <List />
      <Footer />
    </>
  )
}
