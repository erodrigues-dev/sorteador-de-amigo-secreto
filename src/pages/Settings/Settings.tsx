import React from 'react'
import { Card } from '../../components/Card'
import { Footer } from '../../components/Footer'
import Form from '../../components/Form'
import { List } from '../../components/List'

export const Settings = () => {
  return (
    <Card>
      <section>
        <h2>Vamos come√ßar!</h2>
        <Form />
        <List />
        <Footer />
      </section>
    </Card>
  )
}
