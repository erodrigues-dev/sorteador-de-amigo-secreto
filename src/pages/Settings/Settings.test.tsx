import { render } from '@testing-library/react'
import React from 'react'
import { RecoilRoot } from 'recoil'
import { Settings } from './Settings'

jest.mock('react-router-dom', () => ({
  useNavigate: () => jest.fn(),
}))

describe('Pagina de configuração', () => {
  it('deve ser renderizada corretamente', () => {
    const { container } = render(
      <RecoilRoot>
        <Settings />
      </RecoilRoot>
    )

    expect(container).toMatchSnapshot()
  })
})
