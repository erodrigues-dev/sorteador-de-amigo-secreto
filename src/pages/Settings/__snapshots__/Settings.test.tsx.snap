// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagina de configuração deve ser renderizada corretamente 1`] = `
<div>
  <div
    class="card"
  >
    <section>
      <h2>
        Vamos começar!
      </h2>
      <div
        class="card"
      >
        <form>
          <div
            class="input-group-btn"
          >
            <input
              placeholder="Insira os nomes dos participantes"
              type="text"
              value=""
            />
            <button
              disabled=""
              type="submit"
            >
              Adicionar
            </button>
          </div>
        </form>
      </div>
      <ul />
      <footer
        class="footer"
      >
        <button
          class="button"
          disabled=""
        >
          Iniciar
        </button>
        <img
          alt="Sacolas de compras"
          src="/imagens/sacolas.png"
        />
      </footer>
    </section>
  </div>
</div>
`;
